### PR TITLE
Removes lambdaj dependency from code-base

### PR DESCRIPTION
**Pull request by rzo1**
removes lambdaj dependency from code-base

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <name>${project.artifactId}</name>
     <description>Kumo's goal is to create a powerful and user friendly Word Cloud API in Java. Kumo directly generates an image file without the need to create an applet (as many other libraries do).</description>
     <url>https://github.com/kennycason/kumo</url>
-    <version>1.8</version>
+    <version>1.9-SNAPSHOT</version>
```
Comment (kennycason) [inline pom.xml]: Lets just bump this to 1.9

File: src/main/java/com/kennycason/kumo/nlp/FrequencyAnalyzer.java [hunk 2]
```diff
@@ -119,7 +116,7 @@ public FrequencyAnalyzer() {
         allFilters.add(new WordSizeFilter(minWordLength, maxWordLength));
         allFilters.addAll(filters);
         final CompositeFilter compositeFilter = new CompositeFilter(allFilters);
-        return Lambda.filter(compositeFilter, words);
+        return  words.stream().filter(compositeFilter).collect(Collectors.toList());
```
Comment (kennycason) [inline src/main/java/com/kennycason/kumo/nlp/FrequencyAnalyzer.java]: remove extra spacing before words

File: src/main/java/com/kennycason/kumo/nlp/filter/StopWordFilter.java [hunk 3]
```diff
@@ -15,10 +15,10 @@ public StopWordFilter(final Collection<String> stopWords) {
         this.stopWords.addAll(stopWords);
     }
 
+
     @Override
-    public boolean apply(String word) {
-        return !this.stopWords.contains(word.toLowerCase());
+    public boolean test(String s) {
```
Comment (kennycason) [inline src/main/java/com/kennycason/kumo/nlp/filter/StopWordFilter.java]: i'd like to rename the variable back to "word"

File: pom.xml [hunk 4]
```diff
@@ -162,8 +157,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.2</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```
Comment (kennycason) [inline pom.xml]: I have concerns with bumping this to 1.8 immediately. But, meh, lets do it, If this becomes too big of a problem, we can always bring in something like Eclipse Collections, my personal favorite collections library and revert to java 1.7. But I suspect 1.8 bump will be ok.

File: src/main/java/com/kennycason/kumo/nlp/filter/StopWordFilter.java (hunk 3 above)
Comment (rzo1) [inline src/main/java/com/kennycason/kumo/nlp/filter/StopWordFilter.java]: done

File: src/main/java/com/kennycason/kumo/nlp/FrequencyAnalyzer.java (hunk 2 above)
Comment (rzo1) [inline src/main/java/com/kennycason/kumo/nlp/FrequencyAnalyzer.java]: done

File: pom.xml (hunk 4 above)
Comment (rzo1) [inline pom.xml]: Maybe we can include this information as a major change for 1.9 RELEASE in a changelog?

File: pom.xml (hunk 1 above)
Comment (rzo1) [inline pom.xml]: done

**Comment by rzo1** [comment]
I just pushed the related changes

File: pom.xml (hunk 4 above)
Comment (kennycason) [inline pom.xml]: I'll create an "issue" for it.

**Comment by kennycason** [comment]
Thanks for making the changes @rzo1

**Pull request merged**